utils/repos.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+from utils.launch import git_clone
+
+dir_repos = "repositories"
+
+
+def download_repos():
```

Comment:
I created this script because it's obvious that the SD Web UI app needs these repos to exist in the `repositories/` directory.  So, the idea here is that we execute this method on init to verify they exist.